### PR TITLE
create empty project

### PR DESCRIPTION
**Pull request by SolovyovDmitry**
create empty BookingService solution and project for futher development

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN dotnet restore -s https://api.nuget.org/v3/index.json
+
+COPY . ./
+RUN dotnet publish -c Release -o out
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-bullseye-slim AS base
+WORKDIR /app
+COPY --from=build /app/out .
+EXPOSE 80
+ENTRYPOINT ["dotnet", "LT.DigitalOffice.AdminService.dll"]
```
Comment (MariyaSin) [inline Dockerfile:14]: BookingService

File: src/BookingService/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "Seq",
+        "Args": {
+          "serverUrl": "http://localhost:5342",
+          "apiKey": "rOWPuAL2a1Q5Qe44MKCb"
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:20]: apiKey must be unique

File: src/BookingService/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+      }
+    ]
+  },
+  "ServiceInfo": {
+    "Name": "BookingService",
+    "Id": "B2DC617D-99A3-4C77-AD0B-3FFB3619CC9A"
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:27]: do you generate it or move from other service?

File: src/BookingService/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    // CRService
+    "CheckUserRightsEndpoint": "CRService_CheckUserRights",
+
+    // AuthService
+    "ValidateTokenEndpoint": "AuthService_ValidateToken",
+
+    // UserService
+    "CheckUserIsAdminEndpoint": "UserService_GetUserAdminStatus",
+    "CreateAdminEndpoint": "UserService_CreateAdmin",
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:42]: remove

File: src/BookingService/appsettings.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+    // EmailService
+    "CreateSmtpCredentialsEndpoint": "EmailService_CreateSmtpCredentials"
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:45]: remove

File: src/BookingService/appsettings.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  },
+  "CheckTokenMiddleware": {
+    "SkippedEndpoints": [
+      "/apiinformation",
+      "/admin/find",
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:50]: remove

File: src/BookingService/appsettings.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+      "/admin/install",
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:51]: remove

File: src/BookingService/appsettings.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+      "/graphicaluserinterface/get"
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:52]: remove

File: src/BookingService/appsettings.json (hunk 3 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:27]: This one was generated

File: src/BookingService/appsettings.json (hunk 2 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:20]: Where can I get it or generate?

File: Dockerfile (hunk 1 above)
Comment (SolovyovDmitry) [reply Dockerfile:14]: Changed

File: src/BookingService/appsettings.json (hunk 6 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:50]: removed

File: src/BookingService/appsettings.json (hunk 7 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:51]: removed

File: src/BookingService/appsettings.json (hunk 8 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:52]: removed

File: src/BookingService/appsettings.json (hunk 4 above)
Comment (SolovyovDmitry) [reply src/BookingService/appsettings.json:42]: removed

File: src/BookingService/BookingService.csproj [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+	  <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.3" />
+	  <PackageReference Include="Serilog.AspNetCore" Version="5.0.0" />
+	  <PackageReference Include="Serilog.Enrichers.Environment" Version="2.2.0" />
+	  <PackageReference Include="Serilog.Enrichers.Process" Version="2.0.2" />
+	  <PackageReference Include="Serilog.Enrichers.Thread" Version="3.1.0" />
+	  <PackageReference Include="Serilog.Settings.Configuration" Version="3.3.0" />
+	  <PackageReference Include="Serilog.Sinks.Seq" Version="5.1.1" />
+	</ItemGroup>
+
+	<ItemGroup>
```
Comment (MariyaSin) [inline src/BookingService/BookingService.csproj:21]: nugets from 22 to 25 line was include in target file

File: src/BookingService/appsettings.json [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    "Redis": "localhost:6379"
+  },
+  "RabbitMQ": {
+
+    // CRService
+    "CheckUserRightsEndpoint": "CRService_CheckUserRights",
+
+    // AuthService
+    "ValidateTokenEndpoint": "AuthService_ValidateToken"
+
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:39]: add   "CheckTokenMiddleware"

File: src/BookingService/appsettings.json [hunk 11]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (MariyaSin) [inline src/BookingService/appsettings.json:1]: I did not find appsettings.Production.json

File: src/BookingService/appsettings.Production.json [hunk 12]
```diff
@@ -0,0 +1,13 @@
+﻿{
+  "Serilog": {
+  },
+  "ConnectionStrings": {
+    "SQLConnectionString": "Server=dbAdmin-sql-server;User=sa;Password=Admin_1234;Database=AdminDB;"
```
Comment (asyagraf) [inline src/BookingService/appsettings.Production.json:5]: change "admin" to "booking" in SQLConnectionString here and in appsettings.Development.json

**Review by VLiamin — APPROVED** [review]

**Pull request closed without merging**